src/webapp/components/upload/UploadSample.tsx

```diff
@@ -52,10 +52,10 @@ export const UploadSample: React.FC<UploadSampleProps> = ({ batchId }) => {
                         batchId,
                         fileType: SAMPLE_FILE_TYPE,
                     };
-                    const submissionId = await compositionRoot.glassDocuments
+                    const uploadId = await compositionRoot.glassDocuments
                         .upload({ file: uploadedSample, data })
                         .toPromise();
```

Comment:
[not in the PR, for the future] Unless specific causes to justify it, we should use `future.run(sucessBranch, errorBranch)`.

Reply:
Noted in glass team slack, will be addressed in upcoming tasks.